privacyidea/static/components/config/views/config.radius.html

```diff
@@ -10,9 +10,13 @@
                     ng-class="{ active: $state.includes('config.radius.add')}"
                     ng-show="checkRight('radiusserver_write')">
                     <a ui-sref="config.radius.add" translate>
-                        New RADIUS server
-                    </a>
-                </li>
+                        New RADIUS server</a></li>
+                <hr>
+                <li role="presentation">
+                    <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                       privacyideaVersionNumber+'/configuration/radiusserver_config.html#'}}">
```

Comment:
like above

Reply:
done in a37f2d98d2d6876d137ba6a6783b086b289941c3.